server/proto/testdata/merge/1/expected/file1.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+message Person {
```

Comment:
Let's put a comment in expected proto file what's changed. To easily understand test case.